api/db/migrate/20180320110726_create_doorkeeper_tables.rb

```diff
@@ -0,0 +1,69 @@
+class CreateDoorkeeperTables < ActiveRecord::Migration[5.1]
+  def change
+    create_table :oauth_applications do |t|
```

Comment:
@bbonislawski we need those tables with `spree_` prefix - what if someone already has the doorkeeper gem and already have those tables?

Reply:
Ok, I'll take care of it.